elements/pfe-accordion/package.json

```diff
@@ -8,25 +8,35 @@
   "publishConfig": {
     "access": "public"
   },
-  "description": "Accordion element for PFElements",
+  "description": "Accordion for PatternFly Elements",
+  "keywords": [
+    "web-components",
+    "html"
+  ],
   "repository": {
     "type": "git",
-    "url": ""
+    "url": "https://github.com/patternfly/patternfly-elements/tree/v1.0.0-prerelease.10/elements/pfe-accordion"
   },
-  "main": "index.js",
+  "main": "pfe-accordion.js",
   "scripts": {
     "build": "../../node_modules/.bin/gulp && ../../node_modules/.bin/prettier --ignore-path ../../.prettierignore --write '**/*.{js,json}'",
     "dev": "../../node_modules/.bin/gulp dev",
     "test": "../../node_modules/.bin/wct --configFile ../../wct.conf.json elements/pfe-accordion/test/"
   },
-  "author": {
-    "name": "Kyle Buchanan"
-  },
+  "contributors": [
+    {
+      "name": "Kyle Buchanan",
+      "email": "kylebuch8@gmail.com",
+      "url": "https://github.com/kylebuch8"
+    },
```

Comment:
```suggestion
    }, {
      "name": "Chris Doherty"
    }, {
      "name": "Kendall Totten"
    },
```

Reply:
@starryeyez024 I'll get you added too.

Chris had entered himself as `cd` so I wanted to respect that.  Let me know how you want to be cited @chrisdo1.

Reply:
@castastrophe @starryeyez024 just cd is fine. Thanks!